pykokkos/core/runtime.py

```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 
+import kokkos
```

Comment:
From reading through the code, I think there may be an alias or data structure I should use for type access instead of `kokkos` directly?

Reply:
Yeah I've mostly been using `from pykokkos.bindings import kokkos`